it/controllers/benefits/reimbursed/VouchersOrCreditCardsBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,534 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.benefits.reimbursed
+
+import controllers.employment.routes.CheckYourBenefitsController
+import forms.YesNoForm
+import models.User
+import models.benefits.{BenefitsViewModel, ReimbursedCostsVouchersAndNonCashModel}
+import models.mongo.{EmploymentCYAModel, EmploymentDetails, EmploymentUserData}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.HeaderNames
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.WSResponse
+import utils.{EmploymentDatabaseHelper, IntegrationTest, ViewHelpers}
+
+class VouchersOrCreditCardsBenefitsControllerISpec extends IntegrationTest with ViewHelpers with EmploymentDatabaseHelper {
+
+  val taxYearEOY: Int = taxYear - 1
+  val employmentId: String = "001"
+
+  private val userRequest = User(mtditid, None, nino, sessionId, affinityGroup)(fakeRequest)
+
+  private def employmentUserData(isPrior: Boolean, employmentCyaModel: EmploymentCYAModel): EmploymentUserData =
+    EmploymentUserData(sessionId, mtditid, nino, taxYearEOY, employmentId,
+      isPriorSubmission = isPrior, hasPriorBenefits = isPrior, employmentCyaModel)
+
+  def cyaModel(employerName: String, hmrc: Boolean, benefits: Option[BenefitsViewModel] = None): EmploymentCYAModel =
```

Comment:
can remove the employerName and hmrc passed in variables and just set them too "name" and "true" as thats what all the tests set them too anyway

Reply:
Updated as requested.